src/components/ButtonPanel.js

```diff
@@ -5,17 +5,17 @@ export default class ButtonPanel extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      names: ['AC', '+/-', '%', '+',
-        '7', '8', '9', 'X',
-        '4', '5', '6', '-',
-        '1', '2', '3', '+',
-        '0', '.', '='],
+      objects: [{ id: 1, name: 'AC' }, { id: 2, name: '+/-' }, { id: 3, name: '%' },
```

Comment:
State is usually for data that would change during the course of the program, [see this](https://learn.co/lessons/react-updating-state)